src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -641,6 +643,30 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
         paginator.loadPrevious();
     }
 
+    loadFirst = () => {
+        const { paginator } = this.props;
+
+        paginator.loadFirst();
+    }
+
+    loadLast = () => {
+        const { paginator } = this.props;
+
+        paginator.loadLast();
+    }
+
+    lastPage = () => {
+        const { paginator } = this.props;
+
+        return paginator.lastPage;
+    }
+
+    goToPage = (page_number: string) => {
+        const { paginator } = this.props;
+
+        paginator.goToPage(page_number);
```

Comment:
Do the string -> number cast here instead to keep the paginator isolated from the UI details